misc/snapshotter/config.toml

```diff
@@ -10,6 +10,9 @@ enable_system_controller = true
 enable_stargz = false
 # Whether snapshotter should try to clean up resources when it is closed
 cleanup_on_close = false
+# Snapshotter will overwrite daemon's mirrors configuration 
+# if the values loaded from this driectory are not null before starting a daemon.
+mirrors_config_dir = "/etc/nydus/mirrors.d"
```

Comment:
Can we follow the containerd dir naming convention? https://github.com/containerd/containerd/blob/main/docs/cri/config.md#registry-configuration

Reply:
Yes, done.